common/src/main/java/com/github/khanshoaib3/minecraft_access/utils/system/KeyUtils.java

```diff
@@ -50,31 +44,21 @@ public static boolean isAllPressed(int... keyCodes) {
      * our multiple-keybindings-on-one-key usage is not supported.
      */
     public static boolean isAnyPressed(KeyBinding... keyBindings) {
-        return Arrays.stream(keyBindings).anyMatch(b -> {
-                    int keyCode = ((KeyBindingAccessor) b).getBoundKey().getCode();
-                    if (keyCode > 7) {
-                        // If this keybinding is bound to a keyboard-key,
-                        // let's use our key-pressing-check logic to circumvent the limitation.
-                        return KeyUtils.isOnePressed(keyCode);
-                    } else {
-                        // If this keybinding is bound to a non-keyboard key, execute the original method.
-                        return b.isPressed();
-                    }
-                }
-        );
+        return Arrays.stream(keyBindings).anyMatch(isKeyPressed());
```

Comment:
A method reference may be better than returning a lambda.
```suggestion
        return Arrays.stream(keyBindings).anyMatch(this::isKeyPressed);
```